harness/determined/common/api/_util.py

```diff
@@ -20,6 +20,9 @@ class PageOpts(str, enum.Enum):
 # Default max number of times to retry a request.
 MAX_RETRIES = 5
 
+# Seconds to wait for an NTSC task to become ready before timeout.
+NTSC_TIMEOUT = 60 * 30
```

Comment:
bumping to 30 min, though 15 might be enough? not sure if there's any harm though, this is only being used by the CLI and i assume users would kill the process after some time.

Reply:
my only worry is that we have a few tests which use `api.task_is_ready` for test purposes. these may start taking too much time to fail, and for them we'll probably want to have `task_is_ready` accept the timeout as a parameter.

Reply:
good point, updated